src/main/java/com/bouncestorage/swiftproxy/v1/AccountResource.java

```diff
@@ -151,33 +154,45 @@ public BulkDeleteResult bulkDelete(@NotNull @PathParam("account") String account
         }
 
         BulkDeleteResult result = new BulkDeleteResult();
+        Multimap<String, String> removeBlobsMap = ArrayListMultimap.create();
+        List<String> deleteContainers = new ArrayList<>();
         for (String objectContainer : objects) {
-            try {
-                if (objectContainer.startsWith("/")) {
-                    objectContainer = objectContainer.substring(1);
-                }
-                int separatorIndex = objectContainer.indexOf('/');
-                if (separatorIndex < 0) {
-                    blobStore.deleteContainer(objectContainer.substring(1));
-                    result.numberDeleted += 1;
-                    continue;
-                }
-                String container = objectContainer.substring(0, separatorIndex);
-                String object = objectContainer.substring(separatorIndex + 1);
+            if (objectContainer.startsWith("/")) {
+                objectContainer = objectContainer.substring(1);
+            }
+            int separatorIndex = objectContainer.indexOf('/');
+            if (separatorIndex < 0) {
+                deleteContainers.add(objectContainer.substring(1));
```

Comment:
We are probably deleting a non-existent container which does not generate an error.  We should enable/add some swift-tests to check for these behaviors.